### ISSUE TITLE
PD/ SSFA ToR name

### ISSUE THREAD

**Issue by acory** [unicef/etools-issues#428]
Change Menu option and List Title to "PD/SSFA"

**Comment by acory** [comment]
@evrimm @robertavram  is the wording 'Add Programme Document or PD/SSFA' correct ?
shouldn't it be 'Add Programme Document or SSFA' ?

**Comment by evrimm** [comment]
Assuming this is for the button in the header of the list page, it should be "ADD NEW PD/SSFA"

**Comment by acory** [comment]
@evrimm  It's the title of the page on which you  add new PD/SSFA

**Comment by evrimm** [comment]
@acory Yes, you're right, it should be  'Add Programme Document or SSFA'

**Comment by mpawlowski85** [comment]
The avilable screen still says PD/SSFA TOR in both places:
![image](https://user-images.githubusercontent.com/26120371/27292099-511dacb2-54e0-11e7-9ec8-9ba688062a17.png)

**Comment by adi130987** [comment]
This screenshot is from staging. Please check it on demo (https://etools-demo.unicef.org/).
I think this task was moved from develop into staging by mistake. I will not be on staging until pmp-redesign work is finished.

**Comment by roxanavlad** [comment]
@acory , this is done ok on demo environment 
tested on chrome, IE11

**Comment by janpanchalingam1** [comment]
Passed testing
